_posts/2013-01-11-navigate-using-just-your-keyboard.md

```diff
@@ -9,11 +9,26 @@ categories:
   - Quick Tests
 ---
 
-One quick way that you can check your accessibility is to navigate through the site using nothing but your keyboard.
+### Introduction
```

Comment:
It's implied that the paragraph immediately after an `h1` could be introductory. I don't think we need to call this out with a new heading.